src/Collector/ClassNameCollector.php

```diff
@@ -9,22 +9,11 @@
 
 class ClassNameCollector implements CollectorInterface
 {
-
-
     public function getType()
     {
         return 'className';
     }
 
-    private function getRegexByConfiguration(array $configuration)
```

Comment:
Should I move that into another PR then? I thought about this as well but my laziness took over when I opened the PR :P